libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -6,38 +6,58 @@
             [closeOnOutsideClick]="closeOnOutsideClick"
             class="fd-combobox-popover-custom">
     <fd-popover-control>
-        <div class="fd-combobox-control">
-            <fd-input-group [glyph]="glyph"  [compact]="compact" [button]="true" [state]="state" [inShellbar]="inShellbar"
-                            (addOnButtonClicked)="onPrimaryButtonClick()" [buttonFocusable]="buttonFocusable">
-                <input #searchInputElement type="text"
-                       fd-input-group-input
-                       [compact]="compact"
-                       (keydown)="onInputKeydownHandler($event)"
-                       (keyup)="onInputKeyupHandler($event)"
-                       [disabled]="disabled"
-                       [(ngModel)]="inputText"
-                       (ngModelChange)="handleSearchTermChange()"
-                       placeholder="{{placeholder}}">
-            </fd-input-group>
-        </div>
+        <fd-input-group [glyph]="glyph"  [compact]="compact" [button]="true" [state]="state" [inShellbar]="inShellbar"
+                        (addOnButtonClicked)="onPrimaryButtonClick()" [buttonFocusable]="buttonFocusable" [disabled]="disabled">
+            <input #searchInputElement type="text"
+                   fd-input-group-input
+                   [compact]="compact"
+                   (keydown)="onInputKeydownHandler($event)"
+                   (keyup)="onInputKeyupHandler($event)"
+                   [disabled]="disabled"
+                   [(ngModel)]="inputText"
+                   (ngModelChange)="handleSearchTermChange()"
+                   placeholder="{{placeholder}}">
```

Comment:
Be consistent, use brackets.
`[placeholder]="placeholder"`
And if you can group inputs and outputs.